2016/meta/emails/mentor_onboarding.md

```diff
@@ -0,0 +1,7 @@
+Hey %%name%%,
+
+Glad to hear you are interested in mentoring for Hack Camp. Here are the details:
+
+Hack Camp is running from %%camp dates%%, 8:40AM to 6PM. We're hosted at [redacted].
```

Comment:
Are we having them stay for post mortems? Because it'd end at 6:15 and not 6. Also weren't we giving people the option to do before lunch and/or after lunch, just so we can be a bit more flexible?